torch/csrc/distributed/c10d/ProcessGroupNCCL.hpp

```diff
@@ -235,6 +235,28 @@ struct DumpPipe {
 };
 #endif
 
+static constexpr int CoalColl = 0x02, CoalP2P = 0x04;
```

Comment:
why not int CoalColl = 0x01, CoalP2P = 0x02;?

Reply:
Just staying the same as previous values.

Reply:
but since we control it and it is only used within PGNCCL. does it make sense to start with 0x01?